modules/ROOT/pages/architecture.adoc

```diff
@@ -42,6 +42,32 @@ The minimum requirements are 3 controller and 3 worker nodes. 3 controllers enab
 
 Mule applications run on workers. Multiple replicas of applications can run across workers.
 
+== Scaling Considerations
+
+* Required number of controller nodes. The maximum number of controller nodes supported is 3.
+
+** Production environment: 3
+** Non-production environment: 1
+
+* CPU/Memory required for controller nodes
+
+** The default (and required minimum) is 2 cores with 8 GiB memory each. You must consider the amount of resources needed for the internal load balancer when sizing controller nodes. Refer to <link to page with performance numbers???> for additional information.
+
```

Comment:
https://docs.mulesoft.com/runtime-fabric/1.3/deploy-resource-allocation#internal-load-balancer

Reply:
Thank you!